pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pydis_core"
-version = "9.9.1"
+version = "9.9.3"
```

Comment:
likely not worth the version bump here, to reduce conflicts. If you leave the changelog there, and remove the release line, it'll find itself into next release's changelog